src/domain/reservation/manage/filters/ManageReservationsFilters.js

```diff
@@ -40,6 +40,14 @@ class ManageReservationsFilters extends React.Component {
       onSearchChange,
     } = this.props;
 
+    // Ensure that start and end are not empty. Otherwise we'll get an error from respa instead of search results.
+    if (filterName === 'start' && !filters.end) {
+      filters.end = filterValue;
+    }
+    if (filterName === 'end' && !filters.start) {
+      filters.start = filterValue;
+    }
+
```

Comment:
This check should be in DateField onChange handler, not here.
All filter changes, which mostly not related will have to go through those condition check